ui/components/multichain/asset-picker-amount/asset-picker-amount.tsx

```diff
@@ -99,7 +99,7 @@ export const AssetPickerAmount = () => {
       </Box>
       <Box display={Display.Flex}>
         <Text
-          color={TextColor.textAlternative}
+          color={error ? TextColor.errorDefault : TextColor.textAlternative}
```

Comment:
Not a blocker but do you think we should put the color in a variable and then use it?

const textColor = error ? TextColor.errorDefault : TextColor.textAlternative

```suggestion
          color={textColor}
```